[2024-09-13T03:55:42.984+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-13T03:55:43.041+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_python_dependencies_v2.get_matplotlib scheduled__2024-09-07T00:00:00+00:00 [queued]>
[2024-09-13T03:55:43.079+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_python_dependencies_v2.get_matplotlib scheduled__2024-09-07T00:00:00+00:00 [queued]>
[2024-09-13T03:55:43.080+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-09-13T03:55:43.117+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): get_matplotlib> on 2024-09-07 00:00:00+00:00
[2024-09-13T03:55:43.152+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=413) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-13T03:55:43.157+0000] {standard_task_runner.py:72} INFO - Started process 445 to run task
[2024-09-13T03:55:43.175+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python_dependencies_v2', 'get_matplotlib', 'scheduled__2024-09-07T00:00:00+00:00', '--job-id', '553', '--raw', '--subdir', 'DAGS_FOLDER/python_dependencies_dag.py', '--cfg-path', '/tmp/tmppay11ilq']
[2024-09-13T03:55:43.178+0000] {standard_task_runner.py:105} INFO - Job 553: Subtask get_matplotlib
[2024-09-13T03:55:43.359+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_python_dependencies_v2.get_matplotlib scheduled__2024-09-07T00:00:00+00:00 [running]> on host 01a3c7586b71
[2024-09-13T03:55:43.689+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ubeyd' AIRFLOW_CTX_DAG_ID='dag_with_python_dependencies_v2' AIRFLOW_CTX_TASK_ID='get_matplotlib' AIRFLOW_CTX_EXECUTION_DATE='2024-09-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-07T00:00:00+00:00'
[2024-09-13T03:55:43.694+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-13T03:55:48.033+0000] {logging_mixin.py:190} INFO - matplotlib with version: 3.9.2
[2024-09-13T03:55:48.035+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-13T03:55:48.067+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-13T03:55:48.068+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_python_dependencies_v2, task_id=get_matplotlib, run_id=scheduled__2024-09-07T00:00:00+00:00, execution_date=20240907T000000, start_date=20240913T035543, end_date=20240913T035548
[2024-09-13T03:55:48.302+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-13T03:55:48.342+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
